src/menu.js

```diff
@@ -0,0 +1,39 @@
+export default function menuPage() {
```

Comment:
- [ ] It would be great if you use ES6 syntax here. For example, don't export a function before defining a function and use arrow functions.